sequencer/src/state.rs

```diff
@@ -526,19 +522,18 @@ impl ValidatedState {
 
         let mut delta = Delta::default();
 
-        apply_proposal(&mut validated_state, &mut delta, parent_leaf, l1_deposits);
-        // validate_builder(&mut self.fee_merkle_tree, &mut delta, proposed_header)?;
+        apply_proposal(&validated_state, &mut delta, parent_leaf, l1_deposits);
```

Comment:
We need to do something with the return value here! Currently we continue using `validated_state` which is unmodified by this function